cockatrice/src/qt-json/json.cpp

```diff
@@ -269,8 +269,7 @@ QVariant Json::parseObject(const QString &json, int &index, bool &success)
         Json::nextToken(json, index);
 
         //Loop through all of the key/value pairs of the object
-        bool done = false;
-        while(!done)
+        while(true)
```

Comment:
Maybe the solution would be to use a function rather than an infinite loop to get this return value?